docs/concepts/services-networking/service.md

```diff
@@ -154,7 +154,7 @@ than `ExternalName`.
 In Kubernetes v1.0, `Services` are a "layer 4" (TCP/UDP over IP) construct, the 
 proxy was purely in userspace.  In Kubernetes v1.1, the `Ingress` API was added
 (beta) to represent "layer 7"(HTTP) services, iptables proxy was added too, 
-and become the default operating mode since Kubernetes v1.2. In Kubernetes v1.9-alpha,
+and become the default operating mode since Kubernetes v1.2. In Kubernetes v1.8-beta,
```

Comment:
Actually, this version string is confusing even after the change.
I cannot find a tag named `v1.8-beta` on https://github.com/kubernetes/kubernetes/releases/tag/, so many users like me will get even confused by this statement.

Reply:
No Confusion.
Can you find the tag named v1.9-alpha on https://github.com/kubernetes/kubernetes/releases/tag/?
So this version string motions with stage, not refers to a specific version.

Reply:
Wha't your apinion? @haibinxie

Reply:
The exact version number is v1.8.0-beta.0: https://github.com/kubernetes/kubernetes/releases/tag/v1.8.0-beta.0

Reply:
Thanks for checking this. An accurate release number would be helpful.

Reply:
Changed to v1.8.0-beta.0, please check it, thanks.